### DISCUSSION TITLE
How Long Does Package Metadata Persist?

### DISCUSSION THREAD

**Discussion by gfyoung** [verdaccio/verdaccio#2034] [Q&A] [answered]
Related: https://github.com/verdaccio/verdaccio/issues/391

https://registry.verdaccio.org/-/web/detail/jquery

<img width="1243" alt="Screen Shot 2020-12-21 at 7 23 00 PM" src="https://user-images.githubusercontent.com/9273653/102845264-087c1c80-43c2-11eb-9078-5af309432a60.png">

https://www.npmjs.com/package/jquery

<img width="1191" alt="Screen Shot 2020-12-21 at 7 23 18 PM" src="https://user-images.githubusercontent.com/9273653/102845269-0ade7680-43c2-11eb-9c22-3bde4583f762.png">

It's evident that `jQuery` does not have a `4.0.0` (perhaps it was unpublished at some point), yet the [Verdaccio registry](https://registry.verdaccio.org/) continues to claim that there is such a version, hence the question.

**Comment by juanpicado** [comment, marked as answer]
hi @gfyoung please do not rely on https://registry.verdaccio.org/ it is just a registry for builds and is not reliable, it is true there are some versions which does not exist on npmjs, just merely for demo purposes.

**Reply by gfyoung** [reply]
@juanpicado: Thanks for the clarification!
